packages/graph/hash_graph/lib/graph/src/ontology/mod.rs

```diff
@@ -150,10 +163,26 @@ pub struct PersistedLinkType {
     pub identifier: PersistedOntologyIdentifier,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Component)]
+#[serde(rename_all = "camelCase")]
+pub struct LinkTypeTree {
+    pub link_type: PersistedLinkType,
+}
+
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Component)]
 pub struct PersistedEntityType {
     #[component(value_type = VAR_ENTITY_TYPE)]
     #[serde(serialize_with = "serialize_ontology_type")]
     pub inner: EntityType,
     pub identifier: PersistedOntologyIdentifier,
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Component)]
+#[serde(rename_all = "camelCase")]
+pub struct EntityTypeTree {
```

Comment:
I wonder if we want to come up with another name for this, it's not _quite_ a tree, but I guess there are different representations of trees, they don't all have to be linked lists. Just generally if you have a vertex set you also have an edge set so :/

Reply:
I don't really have a suggestion here. We could go with `EntityTypeWithRefs` or `EntityTypeSubGraph`, what do you think?

Reply:
It's a bit verbose but I think I'd be most comfortable with `EntityTypeRootedGraph` or `EntityTypeRootedSubgraph` https://en.wikipedia.org/wiki/Rooted_graph

Reply:
I went with `*RootedSubgraph`: e77e45c419a0388342222696cb2dba835a9b7a11